archive/tar_test.go

```diff
@@ -268,7 +277,6 @@ func testTar(t *testing.T, when spec.G, it spec.S) {
 
 func tarContains(t *testing.T, m string, r func()) {
 	t.Helper()
-	t.Log(m)
```

Comment:
don't like the log statements? I think they can make long tests easier to read and debug

Reply:
These log messages in particular didn't add any value in my opinion since when the assertion fails it specifies exactly where the assertion resides. Since we added 3 additional iterations it simply added noise to the logs.